src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -122,6 +121,71 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
             return Page();
         }
 
+        public async Task<PartialViewResult> OnGetCodeLineSectionAsync(string id, int sectionId, string revisionId = null)
+        {
+            Review = await _manager.GetReviewAsync(User, id);
+            Revision = GetReviewRevision(revisionId);
+            var renderedCodeFile = await _codeFileRepository.GetCodeFileAsync(Revision);
+            var htmlLines = renderedCodeFile.GetCodeLineSection(sectionId);
+            var fileDiagnostics = renderedCodeFile.CodeFile.Diagnostics ?? Array.Empty<CodeDiagnostic>();
+            Comments = await _commentsManager.GetReviewCommentsAsync(id);
+            Lines = CreateLines(fileDiagnostics, htmlLines, Comments, true);
+            TempData["CodeLineSection"] = Lines;
+            TempData["UserPreference"] = PageModelHelpers.GetUserPreference(_preferenceCache, User.GetGitHubLogin()) ?? new UserPreferenceModel();
+            return Partial("_CodeLinePartial", sectionId);
+        }
+
+        public async Task<ActionResult> OnPostToggleClosedAsync(string id)
+        {
+            await _manager.ToggleIsClosedAsync(User, id);
+
+            return RedirectToPage(new { id = id });
+        }
+
+        public async Task<ActionResult> OnPostToggleSubscribedAsync(string id)
+        {
+            await _notificationManager.ToggleSubscribedAsync(User, id);
+            return RedirectToPage(new { id = id });
+        }
+
+        public async Task<IActionResult> OnPostToggleApprovalAsync(string id, string revisionId)
+        {
+            await _manager.ToggleApprovalAsync(User, id, revisionId);
+            return RedirectToPage(new { id = id });
+        }
+
+        public IActionResult OnGetUpdatePageSettings(bool hideLineNumbers = false, bool hideLeftNavigation = false)
+        {
+            _preferenceCache.UpdateUserPreference(new UserPreferenceModel()
+            {
+                UserName = User.GetGitHubLogin(),
+                HideLeftNavigation = hideLeftNavigation,
+                HideLineNumbers = hideLineNumbers
+            });
+            return new EmptyResult();
+        }
+
+        public Dictionary<string, string> GetRoutingData(string diffRevisionId = null, bool? showDocumentation = null, bool? showDiffOnly = null, string revisionId = null)
```

Comment:
showDocumentation is not used. This was done after a recent change from Ben to show/hide documentation at client side.